src/components/NavigationTabs/index.tsx

```diff
@@ -56,6 +56,19 @@ const StyledArrowLeft = styled(ArrowLeft)`
   color: ${({ theme }) => theme.text1};
 `
 
+export function FarmTabs({ active }: { active: 'stable' | '' }) {
```

Comment:
As mentioned in a previous PR, does this need to be `''`, or can it be `null`?  Seems janky to allow `''` but I might not have full context